pq/stm32/setup_kyber.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+mkdir pqm4_kyber512
+mkdir Inc
+cur_dir=$PWD
+cd pqm4_kyber512
+code_dir=$PWD
+cd ../Inc
+inc_dir=$PWD
+
+git clone https://github.com/mupq/pqm4.git /tmp/pqm4
+cd /tmp/pqm4
+git checkout 3bfbbfd30401bd1dce3c497feb2a152713f2e735
```

Comment:
Is it possible to use a tag or branch instead of a commit id?

Reply:
I don't have a tag or branch for that particular commit . I just did a get fetch and that's what I got.

Reply:
I guess I could do a fork and then use my fork and then add a tag name in that fork?  Is that what you mean? But I think it looks more legit when it comes directly from the mupq team.  Let me know your views.

Reply:
You do have a patch, so a fork might be good. A commit id is fine I guess, but might be better to just use their latest master? This is just a suggestion. As-is is acceptable too.

Reply:
I kind of want to leave it as is.  Using their master would be harder to maintain as their code isn't quite stabilized.